FCO.tex

```diff
@@ -105,60 +105,59 @@ \section{Fix mixing ratio vs Pu-equivalent theory}
   \caption{Evolution of the plutonium content in the J1'' and  E3'' stream\label{fig:MW_flow} }
 \end{figure}
 
-The difference on J1'' and E3'' between all 6 calculations can be observed on
-Figure \ref{fig:MW_flow}. First one should only consider in this study the time
-after 15 and 75y, as the calculation need almost 12y to rich an equilibrium. 
+The difference between J1'' and E3'' for all 6 calculations can be observed in
+Figure \ref{fig:MW_flow}. Note that for this study one should only consider the time
+after 15 and 75y, as the calculation needs almost 12y to reach an equilibrium. 
 
-For both stream (J1'' and E3''), the 2 cases without decay are similar in the 2
-calculation methods (W and M). When decay is taking into account, one can
+For both streams (J1'' and E3''), the 2 cases without decay are similar in the 2
+calculation methods (W and M). When decay is taken into account, one can
 observe a small reduction of the plutonium from  J1'' stream as well as a small
 increase of the plutonium from E3'' stream directly due to $^{241}$Pu decay: the E3''
-is not affected much by the decay process as it contains mainly 239Pu, J1''
-stream contains a higher $^{241}$Pu content very sensitive to decay.  The decay tends
-to decrease the reactivity potential of the J1'' stream, which is compensate by
+is not affected much by the decay process as it contains mainly 239Pu, while the J1''
+stream contains a higher $^{241}$Pu content and is very sensitive to decay.  The decay tends
+to decrease the reactivity potential of the J1'' stream, which is compensated by
 the increase of the E3'' stream in the mix.
 
 
 \section{Model enrichment prediction}
 
 The second part of this memo is dedicated to the comparison of the plutonium
```

Comment:
This section talks about "kinds" of models and "categories" of models. I'm not sure if you're using the two words interchangeably to compare two things or if you're comparing two sets of things (4 total?)  If only 2, choose "kind" or "category" and be consistent. If you're comparing 4 combinations then give each one a name and then refer back to it explicitly